### DISCUSSION TITLE
English characters are broken.

### DISCUSSION THREAD

**Discussion by stdust** [WerWolv/ImHex#1216] [Get Help] [answered]
![1123](https://github.com/WerWolv/ImHex/assets/139965217/f7958cf4-5534-4483-b4e8-a5261835c8ba)

As you can see in the picture, the English characters are broken. 
I tried changing some settings, but I couldn't solve it. I need help

**Comment by WerWolv** [comment, marked as answer]
Hey
This is to be expected since you're using a custom font that's not a monospace font. This works for most of the UI but it will never work properly for the hex editor view. The ASCII column, just like the hex column, is designed to have a fixed width and height for every "cell" in order to handle selections and highlighting nice and efficiently. If you want to use a non-monospace font here, each ASCII cell would need to have its own size which will take away a lot of the usefulness of it. 
The same problem technically applies to the hex cells as well, you're mostly just lucky there that the characters 0 - 9 and A - F are all about the same size.

If you don't want to use the default font, I recommend you use one of the [Nerd Fonts](https://www.nerdfonts.com/font-downloads) or any other monospace font

**Reply by stdust** [reply]
I really appreciate your help in resolving the problem. 

Thank you
